doc/source/ray-core/examples/batch_training.ipynb

```diff
@@ -557,13 +591,30 @@
     "    )\n",
     "    load_tasks = [read_into_object_store_pg.remote(file) for file in files]\n",
```

Comment:
You should be able to just run the tasks with `scheduling_strategy="SPREAD"` and not set a placement group.